package.json

```diff
@@ -1,21 +1,34 @@
 {
-  "name": "cms",
-  "private": true,
-  "version": "0.1.0",
-  "description": "A Strapi application",
+  "name": "@cowprotocol/cms",
+  "version": "0.1.0-rc.0",
+  "description": "Cow Protocol CMS",
+  "main": "dist/lib.js",
+  "types": "dist/lib.d.ts",
+  "source": "src/lib.ts",
+  "exports": {
+    "require": "./dist/lib.js",
+    "default": "./dist/lib.js"
+  },
+  "license": "(MIT OR Apache-2.0)",
   "scripts": {
     "dev": "strapi develop",
     "start": "strapi start",
-    "build": "strapi build",
-    "strapi": "strapi"
+    "build": "npm run build:strapi && npm run build:lib",
+    "build:strapi": "npx tsc -p tsconfig.lib.json",
+    "build:lib": "npx tsc -p tsconfig.lib.json",
+    "strapi": "strapi",
+    "types": "npm run types:prod",
+    "types:local": "npx openapi-typescript http://localhost:1337/api/docs/openapi.json -o src/gen/types.ts",
```

Comment:
Deeply sorry for making your life so much harder 😅 , but here we go again

Why reference the file from a webserver? Isn't that json static and shouldn't it be somewhere in this repo?

Please disregard is this makes no sense or takes days to accomplish.

Reply:
Actually, you are right!

It was actually generated in the repository. I thought the Openapi JSON file was dynamically created. But not :)

Reply:
Adapted to make the prepublish to build the library and types from the local file